src/components/TileComponents.js

```diff
@@ -0,0 +1,29 @@
+import { render } from '../shared/dom';
```

Comment:
This should a single, TileComponent
The TileComponents would be an separate component with scroll-like logic

Reply:
I edited code, it renders now a single tile, which contains title and image